editor/plugins/tile_set_editor_plugin.cpp

```diff
@@ -126,8 +126,8 @@ void TileSetEditor::_import_node(Node *p_node, Ref<TileSet> p_library) {
 				Transform2D shape_transform = sb->shape_owner_get_transform(E->get());
 				bool one_way = sb->is_shape_owner_one_way_collision_enabled(E->get());
 
-				shape_transform.set_origin(shape_transform.get_origin() - phys_offset + sb->get_transform().xform(shape_transform.get_origin()));
-
+				shape_transform[2] -= phys_offset - sb->get_transform().xform(shape_transform[2]);
+				
```

Comment:
There's trailing whitespace on this line.

Could you also squash the commits together? See [PR workflow](http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase).

Reply:
Fixed trailing spaces and squashed the commits.